README.md

```diff
@@ -1,7 +1,11 @@
 # Dummy
 
 Dummy makes it clear what elements on the page are "dummy" info (i.e.
-still need to be implemented on the backend).
+still need to be implemented on the backend) by making them visibly different
```

Comment:
According to dictionary.com: 

Visibly: that can be seen; perceptible to the eye

Visually: in a visual manner; with respect to sight; by sight

So visibly makes more sense to me. But @creuter you can be the tie-breaker